### ISSUE TITLE
Images are not being included in question memories

### ISSUE THREAD

**Issue by rbyh** [expectedparrot/edsl#1435]
Images are not getting passed when a memory of a question with an image is created:
https://www.expectedparrot.com/content/d52ef359-be54-4bb3-98ad-5a453035f89c

**Comment by johnjosephhorton** [comment]
What we probably need to do is send the files_list associated with any "remembered" question forward.

**Comment by rbyh** [comment]
Yes - same problem for text. Compare results for a scenario created directly with text vs a scenario created with identical text from a filestore object for a txt file:
https://www.expectedparrot.com/content/2e0801f4-82d7-4a2c-ba11-2ca82a980284

**Comment by rbyh** [comment]
@johnjosephhorton Is this relevant?
`files_list: Optional["Files"] = None,` vs `files_list: Optional[List["FileStore"]] = None,` in `async def async_execute_model_call` for different models:
https://github.com/search?q=repo%3Aexpectedparrot%2Fedsl%20files_list&type=code

**Comment by rbyh** [comment]
Will instead add warning that scenarios for filestore objects are not included in memory rules; they must be added with piping.
